### ISSUE TITLE
Scrape logstash dead letter queue metrics

### ISSUE THREAD

**Issue by Turgon37** [alxrem/prometheus-logstash-exporter#14]
When using log daedletter queue feature https://www.elastic.co/guide/en/logstash/current/dead-letter-queues.html logstash expose DLQ metrics on /_node/stats/pipelines endpoint. Because dead letter queue currently does not support auto-purge, it would be nice that prometheus exporter scrape it size and allow us to set alerting according to maximum configured size.

The metric json key can be found here
https://github.com/elastic/logstash/blob/20f5512103eee552d1f24caeb238cb405a83e19b/logstash-core/spec/logstash/java_pipeline_spec.rb#L1227 

And the definition of the json object here https://github.com/elastic/logstash/blob/06af15030aeeb9122ae4e1ffb4f1b8f7166ef690/logstash-core/lib/logstash/api/commands/stats.rb#L166 

To help you, I've made some curls on my setup

```
server ~ # docker exec -it logstash01 curl localhost:9600/_node/stats/pipelines?pretty | jq '.pipelines.main|keys'
[
  "dead_letter_queue",
  "ephemeral_id",
  "events",
  "hash",
  "plugins",
  "queue",
  "reloads"
] 
```
"dead_letter_queue" key is located next to "plugins" and other known keys. It is not present if the feature is not enabled

Then the content of the json object is 
```
server ~ # docker exec -it logstash01 curl localhost:9600/_node/stats/pipelines?pretty | jq .pipelines.main.dead_letter_queue
{
    "queue_size_in_bytes": 10485733
}
```

**Comment by alxrem** [comment]
Metrics of DLQ was added by #15 and are available in release 0.7.0

**Comment by Turgon37** [comment]
thanks for your work, It's works fine :)
